extensions/broker-server-postgres-flyway-jooq/build.gradle.kts

```diff
@@ -5,7 +5,7 @@ import org.testcontainers.containers.PostgreSQLContainer
 
 val jooqDbType = "org.jooq.meta.postgres.PostgresDatabase"
 val jdbcDriver = "org.postgresql.Driver"
-val postgresContainer = "postgres:11-alpine"
+val postgresContainer = "postgres:15-alpine"
```

Comment:
in the customer portal project I moved this to the libs.versions.toml aswell, and it worked great, fully DRY
 
I don't know how much we can DRY the postgres version here, so please decide if it's worth to move this into the versions.toml

Reply:
DRYed it. It needed a bit of code generation but now there's only 1 instance of that image name.